### DISCUSSION TITLE
Star trails how

### DISCUSSION THREAD

**Discussion by Dentonknifeworks** [aaronwmorris/indi-allsky#821] [Q&A]
Ok I cant seem to get a star trail for the life of me.
I am sure it is because I have not idea what is a good starting point where no matter what I would get star trails
What setting would produce a star trail even if the moon was part of it? I just need a starting point so I can understand the settings and how they work
![Screenshot 2023-07-03 at 11 10 45 AM](https://github.com/aaronwmorris/indi-allsky/assets/64991600/2656bb04-9906-428e-9f1f-79be33cd013e)
I am in a bortal 7 sky

**Comment by aaronwmorris** [comment]
There is a view in ind-allsky that can help determine the max ADU setting.  https://hostname/indi-allsky/adu  This will show you your average ADU in 15 minute blocks at night.

**Keep in mind that you will probably not be able to generate star trails during a full moon**

The `Max ADU` should normally be about 5-10 less than your target ADU for your images, but you can increase this.

I would also set the Star Trails Mask Threshold to be 255 to just disable the setting.  The Mask Threshold and Pixel Threshold are used together.  When 1% (pixel threshold) of pixels exceed Mask Threshold ADU (default 190), then the image is excluded from the star trails.  I have already updated the code with some slightly better descriptions.

**Reply by Dentonknifeworks** [reply]
Ah, that helps a lot.  You should list all of the views available such as this one very handy.
Thanks, we will see what we can come up with now.
